tests/fv/test_multi_host.py

```diff
@@ -0,0 +1,62 @@
+from time import sleep
+import sh
+from sh import docker
+
+
+def multi_host():
```

Comment:
Does this actually get run?  Might be mis-remembering but I thought the nose/unittest framework looked for unittest classes with methods prefixed with test_